runtime/include/mem/jemalloc/chpl-mem-impl.h

```diff
@@ -46,12 +46,30 @@
 
 #define MALLOCX_NO_FLAGS 0
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+extern unsigned get_num_arenas(void);
```

Comment:
I'm sure you already know this so I'm just agreeing with you ...

This is okay in a prototype implementation but for production we definitely don't want to be exposing an unprotected user-namespace global like this.  It should have a `chpl_je_` or similar prefix on it if we're going to retain this architecture, or move back into the .c file as a static otherwise.

Reply:
Right, agreed -- I probably should have changed this even for this PR, but I plan to address it correctly next sprint so if you're not too offended I'll probably leave it as is.

Reply:
It won't kill me for now.   :-)